README.md

```diff
@@ -123,6 +123,20 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: [Join the Discord](https://discord.gg/python), post on an unclaimed Issue with your interest and mention your in-server name.
 - **Notes**:  See this page for their [Reddit announcement](https://www.reddit.com/r/Python/comments/dbj0vh/python_discord_hacktoberfest_2019/). [Contributing Guides](https://pythondiscord.com/pages/contributing/) are available on their website.
 
+### Q
+
+#### **QMK**
+
+- **Swag**: Shirt, Keyboard
+- **Requirements**: Interact 4 times with a QMK repo in any of these ways:
+  - Review/suggest significant changes on a PR that's labelled `hacktoberfest`
+  - Make a comment on an Issue/PR that contributes to the conversation that's labelled `hacktoberfest`
+  - Open a PR that documents any new feature that's consistent with our doc style (check out the `needs doc` label)
+  - Open a PR that translates some documentation
+  - Open a PR that fixes a bug in an issue labelled `hacktoberfest`
```

Comment:
Instead of having a list like this, I'd recommend something like "Requirements are listed on their Hacktoberfest site" with an embedded link. It keeps things cleaner. :smile: